test/controllers/bigbluebutton_api_controller_test.rb

```diff
@@ -537,6 +537,21 @@ class BigBlueButtonApiControllerTest < ActionDispatch::IntegrationTest
     assert_redirected_to encode_bbb_uri('join', server1.url, server1.secret, params).to_s
   end
 
+  test 'redirects user to the corrent join url with only permitted params for join' do
+    server1 = Server.create(url: 'https://test-1.example.com/bigbluebutton/api/',
+                            secret: 'test-1-secret', enabled: true, load: 0)
+    meeting = Meeting.find_or_create_with_server('test-meeting-1', server1)
+    params = { meetingID: meeting.id, password: 'test-password', fullName: 'test-name', test1: '', test2: '' }
+
+    mock_env('JOIN_EXCLUDE_PARAMS' => 'test1, test2') do
```

Comment:
Hmm. The one problem with doing the parsing once at startup is that it means changing the env at runtime might not work.

If we can't get this to work, another option would be to staticly configure some excluded params in the `.env.test` file, just make sure they don't conflict with any used in other tests.